### ISSUE TITLE
creating configuration without a file generate warning

### ISSUE THREAD

**Issue by jvermillard** [eclipse-californium/californium#2142]
If you create a configuration (Cf v3.8) using:
 ```
 var cfg = Configuration.getStandard();
 cfg.set(CoapConfig.MAX_RESOURCE_BODY_SIZE, 200_000);
 ...
 ```
It works but you have these properties files

Now if you try to do it without the file:
```
 var cfg = Configuration.createWithoutFile();
 cfg.set(CoapConfig.MAX_RESOURCE_BODY_SIZE, 200_000);
```

It generates warnings:
```
09:31:49.813 [DefaultDispatcher-worker-23] WARN  o.e.c.elements.config.Configuration {} - Configuration contains no modules, no definitions, and no values!
java.lang.Throwable: stacktrace
	at org.eclipse.californium.elements.config.Configuration.<init>(Configuration.java:502)
	at org.eclipse.californium.elements.config.Configuration.createStandardWithoutFile(Configuration.java:390)
```
From what I read I the configuration framework doc it should work,and it works, there is just an annoying warning stacktrace

**Comment by boaks** [comment]
I will have a look on it.

**Comment by boaks** [comment]
> From what I read I the configuration framework doc it should work,and it works, there is just an annoying warning stacktrace

I guess, you didn't consider to register the modules, you want to use, see [Configuration - Modules](https://github.com/eclipse-californium/californium/blob/main/element-connector/src/main/java/org/eclipse/californium/elements/config/Configuration.java#L141-L149)

**Comment by boaks** [comment]
If you want to use it without prior registering modules, for what ever reasons and what ever the consequences will be, we can add a method to disable this warning. I added it with the stacktrace, because to one or other, which used a framework without deep understanding of that framework, had trouble to know, the execution order of that framework. With that stacktrace it get's clear, where the configuration is called first and if the modules are already registered as expected.

**Comment by boaks** [comment]
Fixes, see PR #2146
